packages/react/src/components/Authenticator/Router/index.tsx

```diff
@@ -1,19 +1,19 @@
 import { CognitoUserAmplify } from '@aws-amplify/ui';
-import * as React from 'react';
 
 import { useAuthenticator } from '..';
 import { View } from '../../..';
 import { ConfirmSignIn } from '../ConfirmSignIn';
 import { ConfirmSignUp } from '../ConfirmSignUp';
 import { ForceNewPassword } from '../ForceNewPassword';
+import { useCustomComponents } from '../hooks/useCustomComponents';
 import { ConfirmResetPassword, ResetPassword } from '../ResetPassword';
 import { SetupTOTP } from '../SetupTOTP';
 import { SignInSignUpTabs } from '../shared';
 import { ConfirmVerifyUser, VerifyUser } from '../VerifyUser';
 
 export type RouterProps = {
   className?: string;
-  children: ({
+  children?: ({
```

Comment:
Glad you made this optional. Always bothered me you had to add the children using `authenticator`